app/controllers/manageiq/graphql/graphql_controller.rb

```diff
@@ -3,6 +3,10 @@
 module ManageIQ
   module GraphQL
     class GraphQLController < ApplicationController
+      include ActionController::HttpAuthentication::Basic::ControllerMethods
```

Comment:
Is this required because we're subclassing from `ActionController::API`? (guessing)

Reply:
Yep! Believe it's included in `ActionController::Base`, but not `API`